chpl/chpl-resources/src/main/resources/email.properties

```
@@ -323,3 +323,15 @@ complaintsReport.filename=Complaints Report
 complaintsReport.subject=Complaints Report
 complaintsReport.heading=Complaints Report
 complaintsReport.paragraph1.body=The report of all complaints is attached.
+
+#Developer Missing Attestatation Change Request Email
+developer.missingAttestationChangeRequest.subject=Semi-annual Attestation Not Submitted
```

```suggestion
developer.missingAttestationChangeRequest.subject=Semi-annual Attestations Not Submitted
```

This seems to be plural everywhere else, including in the DRAFT email text in the ticket